src/main.rs

```diff
@@ -110,7 +114,22 @@ async fn main() -> Result<()> {
             }))
             .await?;
 
-            let mut store = Store::default();
+            let cost_function = |operator: &Operator| -> u64 {
+                match operator {
+                    Operator::LocalGet { .. } | Operator::I32Const { .. } => 1,
+                    Operator::I32Add { .. } => 2,
+                    _ => 0,
+                }
+            };
```

Comment:
The gas schedule goes here. This is just an example copy/pasta'd from the Wasmer middleware repo